pylxd/models/storage_pool.py

```diff
@@ -184,25 +188,27 @@ def save(self, wait=False):
         LXD. An :class:`~pylxd.exceptions.LXDAPIException` will be generated in
         that case.
 
+        :param wait: Whether to wait for async operations to complete.
+        :type wait: bool
         :raises: :class:`pylxd.exceptions.LXDAPIException` if the storage pool
             can't be deleted.
         """
         # Note this method exists so that it is documented via sphinx.
         super().save(wait=wait)
 
-    def delete(self):
+    def delete(self, wait=False):
         """Delete the storage pool.
 
         Implements DELETE /1.0/storage-pools/<self.name>
 
         Deleting a storage pool may fail if it is being used.  See the LXD
         documentation for further details.
-
```

Comment:
For consistency, I think we should keep this empty line separating the free form text from the `:param` part.